CONTRIBUTING.md

```diff
@@ -7,7 +7,7 @@ Welcome to the Zulip community!
 The
 [Zulip community server](https://zulip.com/development-community/)
 is the primary communication forum for the Zulip community. It is a good
-place to start whether you have a question, are a new contributor, are a new
+place to start whether you have a question, as a new contributor, as a new
```

Comment:
The existing text may or may not be ideal, but I think this is more wrong.

Perhaps one way that might be better is:

```diff
- place to start whether you have a question, are a new contributor, are a new
+ place to start whether you have a question, are a new contributor or
```

Reply:
Yeah, @cyphase 's suggestion is a nice improvement. @akashthedeveloper could you please update the PR accordingly?

Reply:
Ok thanks @cyphase for the correction